bundles/org.openhab.binding.dominoswiss/src/main/java/org/openhab/binding/dominoswiss/internal/eGateHandler.java

```diff
@@ -0,0 +1,344 @@
+        if (channelUID.getId().equals(GETCONFIG)) {
+            sendCommand("EthernetGet;\r");
+        }
+    }
+
+    @SuppressWarnings("null")
+    @Override
+    public void initialize() {
+        logger.debug("Initializing the Dominoswiss EGate bridge handler.");
+
+        Configuration config = this.getConfig();
+        host = (String) config.get("ipAddress");
+        port = 1318; // default Port
+        if (!config.get("port").toString().isEmpty()) {
+            port = Integer.parseInt(config.get("port").toString()); // take configured Port
+        }
+
+        if (host != null && port > 0) {
+            // Create a socket to eGate
+            synchronized (this.lock) {
+                try {
+                    egateSocket = new Socket();
+                    egateSocket.connect(new InetSocketAddress(host, port));
+                    egateSocket.setSoTimeout(SOCKET_TIMEOUT);
+                    writer = new BufferedWriter(new OutputStreamWriter(egateSocket.getOutputStream()));
+                    writer.write("SilenceModeSet;Value=0;" + CR);
+                    writer.flush();
+
+                } catch (UnknownHostException e) {
+                    logger.error("unknown socket host {}", host);
+                    try {
+                        egateSocket.close();
+                        updateStatus(ThingStatus.OFFLINE);
+                    } catch (IOException e1) {
+                        logger.warn("EGate Socket not closed {}", e1.toString());
+                    }
+                    egateSocket = null;
+                } catch (SocketException e) {
+                    logger.warn("{}", e.getLocalizedMessage());
+                    try {
+                        egateSocket.close();
+                        updateStatus(ThingStatus.OFFLINE);
+                    } catch (IOException e1) {
+                        logger.warn("EGate Socket not closed {}", e1.toString());
+                    }
+                    egateSocket = null;
+                } catch (Exception e) {
+                    logger.error(
+                            "Error while establishing connection to Dominoswiss eGate Server with Port: {} and IP: {} Error: {}",
+                            port, host, e.toString());
+                    updateStatus(ThingStatus.OFFLINE);
+                }
+                if (egateSocket != null) {
+                    updateStatus(ThingStatus.ONLINE);
+                }
+                startAutomaticRefresh();
+                logger.debug("EGate Handler connected and online, Status {} ", this.getThing().getStatus().toString());
+
+            }
+        } else {
+            logger.warn("Invalid IP address for dominoswiss eGate or wrong port : '{}'/'{}'", host, port);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.OFFLINE.CONFIGURATION_ERROR,
+                    "Cannot connect to dominoswiss eGate gateway. host IP address or port are not set.");
```

Comment:
The log message could be removed, as the status update is already logged by the framework.

The state change originated by updateStatus() is logged to events.log. Including the status detail message.